fmralign/hyperalignment/individualized_neural_tuning.py

```diff
@@ -0,0 +1,150 @@
+from fmralign.alignment_methods import IndividualizedNeuralTuning as BaseINT
+from .regions import compute_searchlights, compute_parcels
+from nilearn.maskers import NiftiMasker
+from nibabel import Nifti1Image
+import numpy as np
+
+
+class IndividualizedNeuralTuning(BaseINT):
```

Comment:
It looks like we don't have an example using this class ; is that right ? If so, it would be useful to add one just to make it a bit more explicit why this subclass is useful. As is, the existing example uses the BaseINT class, which does work with Niimg objects (as described in the docstring).

Otherwise, I have a light preference to not name the class and the subclass with the same name, but if there's a compelling reason, let me know ? I'd imagine we could call this `HA_INT` or something similar.

Reply:
Okay, sounds right ! Indeed we do note have an example with this class as it is just a wrapper, but I don't know which example would be useful. Maybe I shoudl delete it alltogether ?